src/teleop/gui/src/components/Cameras.vue

```diff
@@ -110,8 +101,7 @@ export default {
       });
 
       var request = new ROSLIB.ServiceRequest({
-        primary: this.primary,
-        camera_cmds: msgs,
+        camera_cmds: msg,
```

Comment:
Ok so we actually still want the primary thing, so bring that back

Here's the link to the finalized ChangeCameras.srv:
https://github.com/umrover/mrover-ros/blob/justiny/cameras/srv/ChangeCameras.srv